Training/PythonWebServer/index.html

```diff
@@ -9,6 +9,33 @@
     <script src="/resources/index.js" type="text/javascript"></script>
 </head>
 <body>
-    <h1>Hello world!</h1>
+	<div class='wrapper'>
+        <button class="button" onclick="loadDoc()">UPDATE</button>
+
+    	<div class="display">
+        Time:
+            <p id = "Time"></p> 
+
```

Comment:
The spacing is overall kind of weird in this file. You should try to find an automatic way of fixing it up!